app/src/main/java/fr/free/nrw/commons/upload/categories/CategoriesPresenter.java

```diff
@@ -7,7 +7,7 @@
 import fr.free.nrw.commons.R;
 import fr.free.nrw.commons.category.CategoryItem;
 import fr.free.nrw.commons.repository.UploadRepository;
-import fr.free.nrw.commons.upload.UploadModel.UploadItem;
+import fr.free.nrw.commons.upload.UploadModel;
```

Comment:
Un related

Reply:
Made the necessary change in my next commit